PD-650: Bug fixes in MongoNav (#405)
mongodb/leafygreen-ui: .changeset/seven-masks-deliver.md

---
'@leafygreen-ui/mongo-nav': patch
---

> Enables Cloud SubMenu when account is the actitvePlatform and makes admin prop optional

Reviewer: ```suggestion
Enables Cloud SubMenu when `account` is the `activePlatform`.

 Makes `admin` prop on the `AccountInterface` optional.
```
